packages/azure-kusto-ingest/test/e2eTests/e2eTest.ts

```diff
@@ -11,7 +11,7 @@ import { Client, ClientRequestProperties, KustoConnectionStringBuilder as Connec
 import StreamingIngestClient from "../../src/streamingIngestClient";
 import ManagedStreamingIngestClient from "../../src/managedStreamingIngestClient";
 import { CompressionType, StreamDescriptor } from "../../src/descriptors";
-import { DataFormat, IngestionProperties, JsonColumnMapping, ReportLevel } from "azure-kusto-ingest";
+import { DataFormat, IngestionProperties, JsonColumnMapping, ReportLevel } from "../../src";
```

Comment:
why

Reply:
because azure-kusto-ingest is not an external dependency - note how we import everything else from ../../src

Reply:
Yea im nor sure what’s a better approach for an e2e